package.json

```
@@ -7,11 +7,11 @@
     "build": "npm run readme && grunt build",
     "test": "DIFF_BASE=master ./vendor/xwp/wp-dev-lib/scripts/pre-commit",
     "readme": "./vendor/xwp/wp-dev-lib/scripts/generate-markdown-readme",
-    "phpunit": "DEV_LIB_ONLY=phpunit ./vendor/xwp/wp-dev-lib/scripts/pre-commit",
+    "phpunit": "DEV_LIB_ONLY=phpunit SYNC_README_MD=0 ./vendor/xwp/wp-dev-lib/scripts/pre-commit",
     "preinstall": "composer install",
     "precommit": "DEV_LIB_SKIP=phpunit ./vendor/xwp/wp-dev-lib/scripts/pre-commit",
     "docker-cli": "docker-compose run wpdevlib",
-    "docker-test": "docker-compose run -e DEV_LIB_ONLY=phpunit wpdevlib ./vendor/xwp/wp-dev-lib/scripts/pre-commit",
+    "docker-test": "docker-compose run -e DEV_LIB_ONLY=phpunit SYNC_README_MD=0 wpdevlib ./vendor/xwp/wp-dev-lib/scripts/pre-commit",
```

```suggestion
    "docker-test": "docker-compose run -e DEV_LIB_ONLY=phpunit -e SYNC_README_MD=0 wpdevlib ./vendor/xwp/wp-dev-lib/scripts/pre-commit",
```

I think `docker-compose` expects each individual environment variable to be added with its own `-e` flag:

> Set environment variables (can be used multiple times, not supported in API < 1.25)

https://docs.docker.com/compose/reference/exec/